src/physlearn/utils.py

```diff
@@ -0,0 +1,266 @@
+from datetime import datetime, timedelta
+from hashlib import md5
```

Comment:
most of the functions here are not used (`HdfSource` uses them, and it is migrated to the repository). Need to think if we still want to have them here